packages/vector_graphics/lib/src/listener.dart

```diff
@@ -1,15 +1,31 @@
 import 'dart:ui' as ui;
 import 'dart:typed_data';
 
+import 'package:flutter/rendering.dart';
 import 'package:vector_graphics_codec/vector_graphics_codec.dart';
 
 /// The deocded result of a vector graphics asset.
 class PictureInfo {
   /// Construct a new [PictureInfo].
-  const PictureInfo(this.picture, this.size);
+  PictureInfo._(this._handle, this.size);
```

Comment:
This is making me wonder if we should just delete `decodeVectorGraphics` for now, or at least make it not return the `PictureInfo` type and unexport it.

Reply:
(Doesn't have to be in this PR)

Reply:
Its not exported right now, I think you'd need to import `src` to see it

Reply:
oh wait wrong file, yeah we can move this to src

Reply:
It is exported in vector_grpahics.dart, because we have a public method that returns it there (I added the export)

Reply:
I don't see any public APIs that use PictureInfo, sorry might be blind